core/templates/dev/head/profile/preferences.html

```diff
@@ -176,7 +176,7 @@ <h2 class="oppia-page-heading">Preferences</h2>
               Drag to crop and resize:
               <div class="oppia-profile-picture-crop-area" ng-show="uploadedImage">
                 <button class="btn btn-default oppia-profile-picture-reset-button" ng-click="reset()">
-                  <span class="glyphicon glyphicon-remove"></span>
+                  <i class="material-icons">&#xE92B;</i>
```

Comment:
I think it's better for this to be an "X", not a trashcan. `<i class="material-icons">&#xE14C;</i>` ("clear"), perhaps.